.travis.yml

```diff
@@ -27,7 +27,6 @@ matrix:
       env: PHP_BIN=php-cgi
 
 install:
-  - composer self-update
```

Comment:
[Yes](https://github.com/composer/composer/blob/master/src/Composer/Command/SelfUpdateCommand.php#L136-L139). If you want use this command, you must add into composer.json [disable-tls](https://github.com/composer/composer/blob/master/src/Composer/Command/SelfUpdateCommand.php#L133)